types/index.d.ts

```diff
@@ -5920,6 +5944,29 @@ declare namespace math {
       a2: MathCollection
     ): MathJsChain<T>
 
+    /*************************************************************************
+     * Signal functions
+     ************************************************************************/
+    /**
+     * Compute the transfer function of a zero-pole-gain model.
+     */
+    zpk2tf<T extends number | MathArray | MathArray[]>(
```

Comment:
The idea of the chained functions is that you chain a number and use the chained number as first argument of a function:

```
const result = math.chain(2).add(3).done() // 5
``` 

So, in the chained type definition, the first argument is replaced by the `this` argument, and besides that, all is the same. So it should look something like:

```ts
zpk2tf<T extends MathCollection>(this: MathJsChain<T>, p: T, k?: T): T
```

Reply:
Thanks for the explantion.
I tried to copy the same format of the other functions.
Should it be something like that 
`    zpk2tf<T extends MathCollection>(
      this: MathJsChain<T>,
      z: T,
      p: T,
      k?: T
    ): MathJsChain<T>
    `